openstf/tasks/train_model.py

```diff
@@ -66,6 +68,8 @@ def train_model_task(pj: dict, context: TaskContext) -> None:
         datetime_end=datetime_end,
     )
 
+    context.checkpoint("TimeSeries input")
```

Comment:
Maybe "Retrieved timeseries input" is better here as indicates more accurately where we are in the procces.

Reply:
Good suggestion, I'll add it